src/VCAuthn/ACAPy/Models.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace VCAuthn.ACAPY
+{
+    public class WalletDidPublicResponse
+    {
+        public WalletPublicDid Result { get; set; }
+    }
+    
+    public class WalletPublicDid
+    {
+        [JsonProperty("did")]
+        public string DID { get; set; }
+            
+        [JsonProperty("verkey")]
+        public string Verkey { get; set; }
```

Comment:
Does the spec specify that only one Verkey is supported per DID? @tplooker I know this is a limitation of the Indy library. Do we want to support more than one or is that unnecessary for this work?

Reply:
Not needed anymore. This is unnecessary work.

Reply:
not sure what we are talking about here :)